crates/turbopack-core/src/issue/mod.rs

```diff
@@ -424,7 +423,7 @@ impl IssueSource {
                     } else {
                         SourcePos {
                             line: i - 1,
-                            column: offset - lines[i - 1].bytes_offset,
+                            column: offset - lines[i - 1].bytes_offset - 1,
```

Comment:
This corrects an off-by-one issue. See the updated CJS snapshot test codeframe

Reply:
TBH That doesn't look like the right location for the fix. Why is the `- 1` needed here?

Is `offset` 1-based? (If yes, could we make it 0-based earlier in the chain)
Is `bytes_offset` off by 1? (If yes, could be fix that earlier in the chain)

Reply:
Looks like `offset` (from the ast spans we get from swc) is 1-indexed.